### PR TITLE
close server on terminate of npm run local; improve error logging;  related to #5

### PR DESCRIPTION
**Pull request by brettstack**

### CODE REVIEW

**Comment by brettstack** [comment]
closes #67

**Comment by miguelslemos** [comment]
should update README.md?

**Comment by brettstack** [comment]
@miguelslemos which changes to README do you propose?

File: example/scripts/local.js [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    if (options.cleanup && server && server.close ) {
+      server.close()
+    }
+
+    if (err) console.error(err.stack)
+    if (options.exit) process.exit()
+}
+
+process.on('exit', exitHandler.bind(null, { cleanup: true }))
+process.on('SIGINT', exitHandler.bind(null, { exit: true })) // ctrl+c event
```
Comment (MynockSpit) [inline example/scripts/local.js]: Can/should we attempt to catch a ctrl-z event (SIGTSTP)?

Comment (brettstack) [inline example/scripts/local.js]: Merging; will add this later.

**Pull request merged**